src/librustc/ty/query/plumbing.rs

```diff
@@ -955,9 +966,18 @@ macro_rules! define_queries_inner {
         // predictable symbol name prefix for query providers. This is helpful
         // for analyzing queries in profilers.
         pub(super) mod __query_compute {
```

Comment:
@mw @nnethercote @rust-lang/wg-compiler-performance Have this been used lately or could we just remove this and rely on `-Z self-profile`?

Reply:
I have never (knowingly) used this.

Reply:
I think @nikomatsakis perf-focus tool might have used this. I'm OK with removing it.